padle/padle.py

```diff
@@ -0,0 +1,485 @@
+    #         await self.config.padleToday.set(random.choice(monsters))
+
+    #     await self.config.numDays.set(1)
+    #     await self.config.subs.set([])
+    #     await self.config.allScores.set([])
+    #     await self.config.saveDailyScores.set([])
+    #     allUsers = await self.config.all_users()
+    #     for userid in allUsers:
+    #         user = await self.bot.fetch_user(userid)
+    #         await self.config.user(user).guesses.set([])
+    #         # need to send message if a user is mid-game
+    #         if(await self.config.user(user).start() and not await self.config.user(user).done()):
+    #             await user.send("A full reset occured, the PADle expired.")
+    #         await self.config.user(user).start.set(False)
+    #         await self.config.user(user).done.set(False)
+    #         await self.config.user(user).score.set([])
+    #         await self.config.user(user).editID.set("")
+    #         await self.config.user(user).channelID.set("")
+    #         await self.config.user(user).embedText.set([])
+    #     await ctx.tick()
+
+    # this takes a long time to run
+    # @padle.command()
+    # async def filter(self, ctx):
+    #     dbcog = await self.get_dbcog()
+    #     mgraph = dbcog.database.graph
+    #     final = []
+    #     for i in range(0, 8800):
+    #         monster = await dbcog.find_monster(str(i), ctx.author)
+    #         try:
+    #             nextTrans = mgraph.get_next_transform(monster)
+    #             prevTrans = mgraph.get_prev_transform(monster)
+    #             if(monster is not None and monster.name_en is not None and monster.on_na and
+    #                 ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or
+    #                  "Super Reincarnated" in monster.name_en) and not monster.is_equip and
+    #                     nextTrans is None and monster.level >= 99):
+    #                 final.append(str(i))
+    #         except Exception as e:
+    #             print("Error " + str(i))
+    #             pass
+    #     await ctx.tick()
+    #     print(",".join(final))
+
+    @padle.command()
+    async def guess(self, ctx, *, guess):
+        """Guess a card for the daily PADle"""
+        if ctx.guild is not None:
+            await ctx.send("You can only play PADle in DMs!")
+            return
+        if(not await self.config.user(ctx.author).start()):
+            await ctx.send("You have not started the game of PADle yet, try `^padle start`!")
+            return
+        if(await self.config.user(ctx.author).done()):
+            await ctx.send("You have already played today's PADle!")
+            return
+        dbcog = await self.get_dbcog()
+
+        guessMonster = await dbcog.find_monster(guess, ctx.author.id)
+        if guessMonster is None:
+            await ctx.send("Monster not found, please try again.", delete_after=10)
+            return False
+
+        monsterEmbed = EmbedView(
```

Comment:
You could try doing this with menu2 so that you can get an :x: button for the embed if you want, this would probably be a closable_embed menu which you can import from tsutils, check how padglobal / crowddata have their menu2.